src/Microsoft.AspNetCore.SystemWebAdapters.Authentication/RemoteAuthenticationExtensions.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net.Http;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Authentication;
+
+/// <summary>
+/// Helper methods for registering remote authentication services
+/// </summary>
+public static class RemoteAuthenticationExtensions
```

Comment:
RemoteAuthentication* -> RemoteAppAuthentication* to avoid conflicts with existing types.

Reply:
Done